src/app/index.tsx

```diff
@@ -0,0 +1,125 @@
+import * as React from 'react';
+import { hot } from 'react-hot-loader';
+import './app.scss';
+import * as mac from 'assets/imgs/mac.png';
+import * as windows from 'assets/imgs/windows.png';
+import * as linux from 'assets/imgs/linux.png';
+import * as Secure from 'assets/imgs/icon-shield.svg';
+import * as Offline from 'assets/imgs/icon-offline.svg';
+import * as Yours from 'assets/imgs/icon-hdd.svg';
+import * as OpenSource from 'assets/imgs/icon-github.svg';
+import { Block } from './block';
+import { Platform } from './platform';
+
+const builds = [
+  { name: 'Mac', link: 'https://github.com/MyCryptoHQ/MyCrypto/releases/tag/1.0.0-RC' },
+  { name: 'Windows', link: 'https://github.com/MyCryptoHQ/MyCrypto/releases/tag/1.0.0-RC' },
+  { name: 'Linux', link: 'https://github.com/MyCryptoHQ/MyCrypto/releases/tag/1.0.0-RC' },
+  { name: 'HTML Build', link: 'https://github.com/MyCryptoHQ/MyCrypto/releases/tag/1.0.0-RC' }
```

Comment:
These should be dynamically fetched using the GitHub API.

Reply:
Likewise, we'll need to show 32bit / 64bit links for OSs that make that distinction (No foolproof user agent tricks for this one, unfortunately.)

Side-note to @skubakdj, we might want to switch to `nsis-web` builds for windows to remove the need for a 32 / 64 bit distinction.